draft-cdn-control-header.md

```diff
@@ -60,9 +60,9 @@ See also the draft's current status in the IETF datatracker, at
 
 # Introduction
 
-Many HTTP origin servers use gateway caches to speed up distributing their content, either locally (sometimes called a "reverse proxy cache") or in a distributed fashion ("Content Delivery Networks").
+Many HTTP origin servers Content Delivery Networks (i.e., distributed HTTP gateways, usually implementing caches) to speed up distributing their content.
```

Comment:
I think you mean to say "server use Content Delivery Networks..."

Reply:
```suggestion
Many HTTP origin servers use Content Delivery Networks (i.e., distributed HTTP gateways, usually implementing caches) to speed up distributing their content.
```